app/dot/cpp/core/services/LoginService.java

```diff
@@ -127,26 +126,14 @@ private String getAccessToken(String userId) {
             .setSubject(userId)
```

Comment:
How about setting the subject to something as AUTH_TOKENS and inside the claim both userId and userRole. It's just an idea that popped into my head, that way when retrieving them it would be directly by the name

Reply:
Set the subject "AUTH_TOKENS" for one access token? Why waste a field with a useless text? The way I see it is, this token has this user as the subject, and for this user I have these claims.